src/service/hooks/useFetch.ts

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from "react";
```

Comment:
Not sure what this component adds vs using fetch() direct in code. I see it's mostly commented-out in other files except for SideMenu. Is it needed?